schema/field/field_test.go

```diff
@@ -164,3 +171,35 @@ func TestField_UUID(t *testing.T) {
 	assert.NotNil(t, fd.Default)
 	assert.NotEmpty(t, fd.Default.(func() uuid.UUID)())
 }
+
+func TestTypeString(t *testing.T) {
+	var typ = field.TypeBool
+	assert.Equal(t, "bool", typ.String())
+	typ = field.TypeInvalid
+	assert.Equal(t, "invalid", typ.String())
+	typ = 21
+	assert.Equal(t, "invalid", typ.String())
+}
+
+func TestTypeNumeric(t *testing.T) {
+	var typ = field.TypeBool
+	assert.Equal(t, false, typ.Numeric())
+	typ = field.TypeUint8
+	assert.Equal(t, true, typ.Numeric())
+}
+
+func TestTypeValid(t *testing.T) {
+	var typ = field.TypeBool
+	assert.Equal(t, true, typ.Valid())
+	typ = 0
+	assert.Equal(t, false, typ.Valid())
+}
+
+func TestTypeConstName(t *testing.T) {
+	var typ = field.TypeJSON
```

Comment:
ditto

Reply:
I should follow the code style in  `ent`.